src/pages/Map.css

```diff
@@ -0,0 +1,70 @@
+.PinCard {
+  width: 250px;
+  height: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.PinCardLabel {
+  width: max-content;
+  color: pink;
+  font-size: 15px;
```

Comment:
Please use `rem` for font-sizes instead of pixels.

https://www.sitepoint.com/understanding-and-using-rem-units-in-css/